stl/inc/chrono

```diff
@@ -1944,65 +1944,75 @@ namespace chrono {
             return _Elapsed_offset;
         }
 
-    private:
-        sys_seconds _Date;
-        bool _Is_positive;
-        seconds _Elapsed_offset;
-    };
+        _NODISCARD friend constexpr bool operator==(const leap_second& _Left, const leap_second& _Right) noexcept {

```

Comment:
No change requested: I don't think we need a comment here to indicated speculative implementation of LWG-4139, despite that this is a fairly big change and that LWG hasn't yet signed off on a proposed resolution. LWG _has_ demonstrated approval of the "make `leap_second`'s operators hidden friends" approach, and both libc++ and libstdc++implement it, so I have no doubt this will be the eventual resolution of the issue.